### PR TITLE
P1/fix/rename badge to achievement

### PR DESCRIPTION
**Pull request by netlight-johannadagfalk**
Badge is still a name in the database... What should we do about that?

### CODE REVIEW

**Review by netlightebba — APPROVED** [review]
🦘

File: app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionContainer.jsx [hunk 1]
```diff
@@ -190,14 +190,12 @@ const KanbanActionContainer = ({
     });
   };
 
-  const orderBadgesOnItemDragged = (performedCategories, movedItem) => {
-    return orderBy(performedCategories, ({ id }) =>
+  const orderAchievementsOnItemDragged = (achievement, movedItem) => {
```
Comment (elinaolsson) [inline app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionContainer.jsx:193]: The previous name "performedCategories" implies that there are several, is it one or several achievements?

Comment (elinaolsson) [reply app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionContainer.jsx:193]: "achievement" or "achievements"?

**Review by elinaolsson — APPROVED** [review]
Gr8 job! 😎

**Pull request merged**